src/containers/ClusterTasks/ClusterTasks.js

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import {
+  InlineNotification,
+  StructuredListBody,
+  StructuredListCell,
+  StructuredListHead,
+  StructuredListRow,
+  StructuredListSkeleton,
+  StructuredListWrapper
+} from 'carbon-components-react';
+
+import Information16 from '@carbon/icons-react/lib/information/16';
+import { fetchClusterTasks } from '../../actions/tasks';
+import {
+  getClusterTasks,
+  getClusterTasksErrorMessage,
+  isFetchingClusterTasks
+} from '../../reducers';
+
+import '../../components/Definitions/Definitions.scss';
+
+export /* istanbul ignore next */ class ClusterTasksContainer extends Component {
+  componentDidMount() {
+    this.props.fetchClusterTasks();
+  }
+
+  render() {
+    const { error, loading, clusterTasks } = this.props;
+
+    return (
+      <>
+        {(() => {
```

Comment:
I think I introduced this pattern originally as part of some incomplete refactoring, but we should really remove it.

Instead of wrapping everything in `<>{(() => { ... })}</>` we could just have the contents directly

```js
render() {
  const { error, loading, clusterTasks } = this.props;

  if (loading && !clusterTasks.length) {
    return <StructuredListSkeleton border />;
  }

  ...
}
```

Some of the containers have already been updated but I'll create an issue to address the remaining ones.